### PR TITLE
Occitan version update

### PR DESCRIPTION
**Pull request by Quent-in**
| Q             | A
| ------------- | ---
| Bug fix?      | yes|no
| New feature?  | yes|no
| BC breaks?    | yes|no
| Deprecations? | yes|no
| Tests pass?   | yes|no
| Documentation | yes|no
| Translation   | yes|no
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

to be ready for the future of Wallabag :)

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 1]
```diff
@@ -344,26 +344,26 @@ import:
...
+       how_to: "Mercés de causir lo fichièr de salvagarda e de clicar sul boton dejós per l'importar. Notatz que lo tractament pòt durar un moment ja que totes los articles an d'èsser recuperats."
     instapaper:
-        page_title: 'Importer > Instapaper'
-        # description: 'This importer will import all your Instapaper articles. On the settings (https://www.instapaper.com/user) page, click on "Download .CSV file" in the "Export" section. A CSV file will be downloaded (like "instapaper-export.csv").'
-        # how_to: 'Please select your Instapaper export and click on the below button to upload and import it.'
+        page_title: 'Importar > Instapaper'
+        description: 'Aquesta aisina importarà totas vòstres articles d'Instapaper. Sus la pagina de paramètres (https://www.instapaper.com/user), clicatz sus "Download .CSV file" dins la seccion "Export". Un fichièr CSV serà telecargat (aital "instapaper-export.csv").'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: The single quote `'` (in `d'Instapaper`) is breaking the file.
Can you do in an other way?

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 2]
```diff
@@ -344,26 +344,26 @@ import:
...
         how_to: "Mercés de seleccionar vòstre Readability fichièr e de clicar sul boton dejós per lo telecargar e l'importar."
     worker:
-        # enabled: "Import is made asynchronously. Once the import task is started, an external worker will handle jobs one at a time. The current service is:"
+        enabled: "L'importacion se fa de manièra asincròna. Un còp l'importacion lançada, una aisina externa s'ocuparà dels messatges un per un. Lo servici actual es : "
     firefox:
-       page_title: 'Importer > Firefox'
-       # description: "This importer will import all your Firefox bookmarks. <p>For Firefox, just go to your bookmarks (Ctrl+Maj+O), then into \"Import and backup\", choose \"Backup...\". You will obtain a .json file."
-       # how_to: "Please choose the bookmark backup file and click on the button below to import it. Note that the process may take a long time since all articles have to be fetched."
+       page_title: 'Importar > Firefox'
+       # description: "Aquesta aisina importarà totas vòstres favorits de Firefox. <p>For Firefox, just go to your bookmarks (Ctrl+Maj+O), then into \"Import and backup\", choose \"Backup...\". You will obtain a .json file."
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: Do not forget to remove the comment `# `

**Comment by Quent-in** [comment]
Sorry I wasn't free last night to fix theses changes :S 
Thanks for having made them!

**Comment by j0k3r** [comment]
👍

**Pull request merged**